esp/src/src/data/components.ts

```diff
@@ -2,67 +2,83 @@ export const DATA = {
     "components": [
         {
             name: "Activity",
-            widget: "ActivityWidget"
+            widget: "ActivityWidget",
```

Comment:
At some point we may want to drop the "widget" and let the routes dictate the actual widget?